### PR TITLE
Add release plan for nightly release

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

File: .konflux/next/release-plan.yaml [hunk 1]
```diff
@@ -8,29 +8,17 @@ metadata:
...
-          repository: "quay.io/openshift-pipelines/openshift-pipelines-pipelines-triggers-eventlistenersink-rhel8"
-          tags: [next]
   tenantPipeline:
+    serviceAccountName: release-registry-openshift-pipelines-dev
     pipelineRef:
       resolver: git
       params:
         - name: url
-          value: "https://github.com/konflux-ci/release-service-catalog.git"
+          value: https://github.com/savitaashture/tenant-pipelines.git
```
Comment (vdemeester) [inline .konflux/next/release-plan.yaml:17]: we'll probably want to move this in `openshift-pipelines/hack` I think.

**Review by vdemeester — COMMENTED** [review]
In the near future, those things should probably be either generated by `hack` or be in `hack` (especially as the SA, secret and role will be shared between all application in the tekton-ecosystem tenant).

File: .konflux/next/release-plan.yaml (hunk 1 above)
Comment (savitaashture) [reply .konflux/next/release-plan.yaml:17]: Yes make sense
will add it there

**Comment by savitaashture** [comment]
Created hack PR https://github.com/openshift-pipelines/hack/pull/38 
which contains Pipeline and required role, sa and secret

**Comment by vdemeester** [comment]
Closed by #80

**Pull request closed without merging**